org.codeberg.dnkl.foot.yml

```diff
@@ -0,0 +1,45 @@
+app-id: org.codeberg.dnkl.foot
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: /app/bin/foot.sh
+
+finish-args:
+  - --socket=wayland
```

Comment:
what happens if the user doesn't have Wayland?

Reply:
If you take a lookj at the [linked Codeberg repo](https://codeberg.org/dnkl/foot) you will find this description:

>  A fast, lightweight and minimalistic Wayland terminal emulator

so it looks like it is Wayland only

Reply:
yes, it's wayland only

Reply:
This should be mentioned in the appdata.